app/services/govpay_refund_details_service.rb

```diff
@@ -22,9 +22,13 @@ def run(refund_id:)
 
   attr_reader :payment, :refund
 
+  def defra_ruby_govpay_api
+    DefraRubyGovpay::API.new(host_is_back_office: true)
```

Comment:
I don't think we want new govpay API connection to be created on each request. Probably worth storing and reusing existing connection (singleton)

Reply:
As discussed - the govpay gem creates a new request each time so there would be little or no advantage to reusing the API object.